docs/loxcall/index.md

```diff
@@ -0,0 +1,74 @@
+# Loxcall by Smarteon
+
+Loxcall is a free smartphone app that enables you to connect your phone calls to your loxone smart home. 
```

Comment:
- [ ] As Loxone is a name, let's put upper `L` everywhere.
- [ ] s/smartphone/android phone ?